test/ngAnimate/animateCssSpec.js

```diff
@@ -28,9 +35,19 @@ describe("ngAnimate $animateCss", function() {
 
   var ss, prefix, triggerAnimationStartFrame;
   beforeEach(module(function() {
-    return function($document, $window, $sniffer, $$rAF, $animate) {
+    return function($document, $sniffer, $$rAF, $animate) {
       prefix = '-' + $sniffer.vendorPrefix.toLowerCase() + '-';
-      ss = createMockStyleSheet($document, $window);
+      ss = createMockStyleSheet($document);
+      ss.addPossiblyPrefixedRule = function(selector, styles) {
```

Comment:
any reason why this is not done here
https://github.com/angular/angular.js/pull/13686/files#diff-40c3b4c40fc4d81bed628855f8d24f26R67 ?